### PR TITLE
update to latest schema

### PR DESCRIPTION
**Pull request by ultrasaurus**
also removed reference to jeremiakimelman

### CODE REVIEW

**Comment by ultrasaurus** [comment]
@jeremiak I would guess this file was just missed when we did the save-as-draft feature, since we all had existing databases that we upgraded.  I discovered the issue of missing `submittedAt` when I cleaned by db and started fresh, for testing.

**Comment by jeremiak** [comment]
Good catch, @ultrasaurus

File: tools/postgres/schema/current.sql [hunk 1]
```diff
@@ -856,6 +856,7 @@ CREATE TABLE task (
     "updatedAt" timestamp with time zone,
     "deletedAt" timestamp with time zone,
     "publishedAt" timestamp with time zone,
+    "submittedAt" timestamp with time zone,
```
Comment (rogeruiz) [inline tools/postgres/schema/current.sql]: aha, I see. so does having this column in the DB fix the issue #1260 @ultrasaurus?

**Comment by ultrasaurus** [comment]
@rogeruiz it fixed the bug, but I wasn't able to make a test that failed before this fix... was trying to test drive the fix so if it ever happened again the tests would catch it

**Pull request merged**